src/Neo/Persistence/IReadOnlyStore.cs

```diff
@@ -32,7 +32,7 @@ public interface IReadOnlyStore
         /// </summary>
         /// <param name="key">The key of the entry.</param>
         /// <returns>The data of the entry. Or <see langword="null"/> if it doesn't exist.</returns>
-        /// <seealso cref="TryGet(byte[], out byte[])"/>. Obsolete it later for avoiding complier warning.
+        [Obsolete("use TryGet(byte[] key, out byte[] value) instead.")]
```

Comment:
can you add for `TryGet` on the `out` parameter; add attribute `NotNullWhen(true)`

Reply:
Looks like it already committed.

Reply:
I was talking about the C# analyzers.